README.md

```diff
@@ -1,27 +1,38 @@
-# Project Name
+# FCC SURVEY FORM
 
-> One paragraph statement about the project.
+> A survey form used for generating data.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description of the project and its features.
-
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML5 form elements
+  - `h1`, `select`, `form`, `button`, `div`, `span`
+  - `textarea`, `label`, `ìnput`, `option`, `fieldset`, .
+- CSS properties
+  - CSS Background for page aesthetic using `background-color`, `background-image`,
+    `box-shadow`, `background-position`,
+    `background-repeat`
+    `text-shadow`, etc.
```

Comment:
Take a look at the background, box-shadow and text-shadow does not belong there they are not background property